Phocalstream_Web/Views/Home/Index.cshtml

```diff
@@ -128,7 +164,7 @@
     </div>
     
     <div class="background">
-        <img class="background-img" src="/api/photo/high/@Model.Sites.ElementAt(Model.SiteIndex).CoverPhotoID" alt="@Model.Sites.ElementAt(Model.SiteIndex).SiteName">
+        <img class="background-img" src="/api/photo/auto/@Model.Sites.ElementAt(Model.SiteIndex).CoverPhotoID" alt="@Model.Sites.ElementAt(Model.SiteIndex).SiteName">

```

Comment:
@icotting, we probably want to set this to be `high` regardless of the connection speed. The full screen background image looks pretty blurry at any other resolution.